packages/react/src/__tests__/ReactContextValidator-test.js

```diff
@@ -263,13 +263,13 @@ describe('ReactContextValidator', () => {
 
     class Component extends React.Component {
       render() {
-        return <TestContext.Provider />;
+        return <TestContext.Provider value={null} />;
```

Comment:
Make this `value={undefined}` instead and you won't need to change the message.

Reply:
now using undefined and reverted message